docs/UserGuide.md

```diff
@@ -334,20 +342,35 @@ to add a meeting from 1330 to 1430 when you already have one scheduled for
 
 #### 5.2.2 List meetings: `listMeeting`
 
-Show the list of meetings in MyInsuRec.
-If used with the optional parameter `[d/DATE]`, *MyInsuRec* will show only meetings happening in that time period.
+Shows a list of meetings in MyInsuRec.
 
 Format: `listMeeting [d/DATE]`
 
+<div markdown="span" class="alert alert-info">:information_source: **Note:**
+* You can use the `d/DATE` parameter optionally to view the list of meetings happening in that time period!
 * `DATE` is specified by keywords. The possible keywords are:
-  * `tomorrow` for a list of meetings happening today or tomorrow;
-  * `week` for a list of meetings happening today or in the next 7 days;
-  * `month` for a list of meetings happening this month.
+  * `tomorrow` for a list of meetings happening tomorrow;
+  * `week` for a list of meetings happening in the next week;
+  * `month` for a list of meetings happening in the next month.
+* For example, `listMeeting d/week` will show a list of meetings happening in the next week.
+* This is an excellent feature if you want to get a quick overview of your upcoming schedule!
+</div>
 
-Examples:
-* `listMeeting`
-* `listMeeting d/tomorrow`
-* `listMeeting d/month`
+<div markdown="span" class="alert alert-success">:exclamation: **Tips and tricks:**
+This command is used to get the index of a meeting. In order to perform commands related to a particular meeting such as `editMeeting`, you will have to first get its index by running `listMeeting`. So, expect to use this command a lot!
+</div>
+
+Example inputs:
+1. `listMeeting`
+2. `listMeeting d/month`
+
+Expected behavior:
+1. Displays **a list of all the meetings**.
+2. Displays **a list of meetings happening in the next month**.
+
+Use case:
+1. Get an overview of all your upcoming meetings. This is especially useful for if you have a busy and packed schedule, and want to ease your mental load of having to recall all of your upcoming meetings!
+2. Organize all your meetings in a single place, so you don't have to worry about missing a meeting ever again.
```

Comment:
Would prefer the use case to be at the top rather than the bottom so the user knows what the commands does before learning about the command

Reply:
sure, i'll find a suitable location for them closer to the top

Reply:
ill put them right after the format